WordPress/Classes/Stores/StatsInsightsStore.swift

```diff
@@ -955,6 +955,17 @@ extension StatsInsightsStore {
         return state.publicizeFollowers
     }
 
+    /// TODO - refactor other functions to use this however cannot find a valid site where publicize > 0 (yet)
```

Comment:
Do we need to do this before merging?

Reply:
Thanks for reminding me - I've committed a change for this